docs/guides/building/windows.md

```diff
@@ -250,9 +264,11 @@ Note that `ComponentGroup`, `Component`, `FeatureGroup`, `Feature` and `Merge` e
 
 ## Internationalization
 
-The Windows Installer is built using the `en-US` language by default. Internationalization (i18n) can be configured using the [`tauri.bundle.windows.wix.language`] property, defining the languages Tauri should build an installer against. You can find the language names to use in the Language-Culture column on [Microsoft's website][localizing the error and actiontext tables].
+The NSIS Installer is a multi-language installer, which means you always have a single installer which contains all the selected translations. You can specify which versions to include using the [`tauri.bundle.windows.nsis.languages`](../../api/config.md#nsisconfig.languages) property.
```

Comment:
we should add a note here about missing languages and redirect users to open a PR

Reply:
True, wanted to keep that really short for now and just link to the config prop but i'll at least at that one. On that topic, i think we linked the wrong one in the config docs, we're [linking to the nsis repo](https://github.com/tauri-apps/tauri/blob/dev/core/tauri-utils/src/config.rs#L460) instead of our own (https://github.com/tauri-apps/tauri/tree/dev/tooling/bundler/src/bundle/windows/templates/nsis-languages). Or was that intentional and i'm misunderstanding the intention here?

Reply:
Does this work? (preview still building...)

Reply:
sounds good 👍